sdk/azidentity/default_azure_credential_test.go

```diff
@@ -21,11 +21,10 @@ func TestDefaultAzureCredential_ExcludeEnvCredential(t *testing.T) {
 		t.Fatalf("Did not expect to receive an error in creating the credential")
 	}
 
-	if len(cred.sources) != 1 {
-		t.Fatalf("Length of ChainedTokenCredential sources for DefaultAzureCredential. Expected: 1, Received: %d", len(cred.sources))
+	if len(cred.sources) != 2 {
```

Comment:
Probably replace the `2` with a constant so when another credential is added to the chain you only have to update the test in one place.

Reply:
Done